mockld/callback_helpers.go

```diff
@@ -0,0 +1,60 @@
+package mockld
+
+import (
+	"bytes"
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/launchdarkly/sdk-test-harness/framework"
+
+	"github.com/gorilla/mux"
+)
+
+type callbackService struct {
```

Comment:
This provides a simple callback service structure that we can reuse for any other instrumented components that the test service might want to provide, if they have multiple endpoints and can take parameters, as in the Big Segment store. For instance, I think we could create cross-platform tests for the persistent data store feature in the same way.